kube-client/src/config/mod.rs

```diff
@@ -287,10 +287,10 @@ impl Config {
         options: &KubeConfigOptions,
     ) -> Result<Self, KubeconfigError> {
         let loader = ConfigLoader::new_from_kubeconfig(kubeconfig, options).await?;
-        Self::new_from_loader(loader).await
+        Self::new_from_loader(loader)
     }
 
-    async fn new_from_loader(loader: ConfigLoader) -> Result<Self, KubeconfigError> {
+    fn new_from_loader(loader: ConfigLoader) -> Result<Self, KubeconfigError> {
```

Comment:
this is technically breaking, and we have avoided changing this in the past due to us potentially loading files via tokio::fs.

i guess it is required for the try_from though, so it's probably worth it.

Reply:
All I could find is https://github.com/kube-rs/kube/discussions/1336, and I think in this instance it is a private method signature change, so it shouldn't be? Since it is not a part of any trait and used only internally? My thought on this was that in case async is needed later, it can be duplicated in a separate method.